pom.xml

```diff
@@ -264,13 +264,19 @@
             <plugin>
                 <groupId>se.eris</groupId>
                 <artifactId>notnull-instrumenter-maven-plugin</artifactId>
-                <version>0.6.8</version>
+                <version>1.0.0</version>
                 <executions>
                     <execution>
                         <goals>
                             <goal>instrument</goal>
                             <goal>tests-instrument</goal>
                         </goals>
+                        <configuration>
+                            <notNull>
+                                <param>org.jetbrains.annotations.NotNull</param>
+                                <param>javax.validation.constraints.NotNull</param>
+                            </notNull>
+                        </configuration>
```

Comment:
Still not really sure why this is needed. Maybe one of our dependencies uses `javax.validation.constraints`.

Reply:
Yea probably